packages/react-native-renderer/src/ReactNativeRenderer.js

```diff
@@ -41,7 +40,7 @@ ReactFiberErrorLogger.injection.injectDialog(
   ReactNativeFiberErrorDialog.showDialog,
 );
 
-const ReactNativeFiber: ReactNativeType = {
+const ReactNativeRenderer: ReactNativeType = {
```

Comment:
It's a bit confusing that we now have `ReactNativeRenderer` that requires a file named `ReactNativeFiberRenderer`.

Reply:
Agree. I’m also not over the moon with arbitrary ReactDOM/ReactDOMComponent separation. Need to do a pass restructuring those later.

Reply:
Yeah, it's def not a blocker to this PR. Just an observation that the naming seems even weirder now 😄